sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpResponseBase.java

```diff
@@ -60,7 +60,7 @@ public final HttpHeaders getHeaders() {
     private static HttpHeaders fromOkHttpHeaders(Headers headers) {
```

Comment:
Should we make an `HttpHeaders` implementation for OkHttp as well?

Reply:
I've made this code far smaller (by using a built-in feature to create a multimap in okhttp), but it wasn't in my plans to do this, simply because okHttp isn't on our critical path for perf. It could be followed up in the future though.